tensorflow/core/grappler/optimizers/auto_mixed_precision_test.cc

```diff
@@ -112,19 +112,33 @@ class AutoMixedPrecisionTest : public GrapplerTest {
     if (gpu_available_) {
       virtual_cluster_.reset(new SingleMachine(/* timeout_s = */ 10, 1, 1));
     } else {
-      DeviceProperties device_properties;
-      device_properties.set_type("GPU");
+      if( num_gpus > 0) {
```

Comment:
Please use `else if (num_gpus > 0) {` instead. This way we can reduce 1 nested `if`.

Reply:
Thanks for the review. I have addressed your comments.